pkgs/development/libraries/qt-5/modules/qttools.nix

```diff
@@ -11,6 +23,12 @@ qtModule {
         -e '/^cmake_linguist_config_version_file.input =/ s|$$\[QT_HOST_DATA.*\]|${lib.getDev qtbase}|'
     sed -i "src/qtattributionsscanner/qtattributionsscanner.pro" \
         -e '/^cmake_qattributionsscanner_config_version_file.input =/ s|$$\[QT_HOST_DATA.*\]|${lib.getDev qtbase}|'
+    sed -i "src/qdoc/qdoc.pro" \
+        -e '/^cmake_qdoc_config_version_file.input =/ s|$$\[QT_HOST_DATA.*\]|${lib.getDev qtbase}|'
```

Comment:
why this is not part of `lib.optionalString withClang`  so we can rebuild 0 ?

Reply:
Low rebuild amount isn't really a concern for me with this PR. We haven't had Qt5 `qdoc` in a long time, I don't need this available ASAP, and none of my Lomiri packages have been written with a functioning `qdoc` in mind anyway.

In other packages, hiding non-conditional patches behind conditionals is usually frowned upon AFAICT. Increases risk of bitrotting when the patching only happens *sometimes*, or even *rarely*.

Reply:
Moved to a patch where we `substituteAll` the `lib.getDev qtbase` reference.